code-challenges.test.js

```diff
@@ -14,56 +14,177 @@
+    // Make 3 expects for the 3 const varaibles procided. 
+describe("divisibleOrNot", () => {
+    const num1 = 15
+    const num2 = 0
+    const num3 = -7
+    it("returns whether the number is evenly divisible by three or not", () => {
+        expect(divisibleOrNot(num1)).toEqual("15 is divisble by three")
+        expect(divisibleOrNot(num2)).toEqual("0 is divisble by three")
+        expect(divisibleOrNot(num3)).toEqual("-7 is not divisble by three")
+    })
+})
 
 
 // b) Create the function that makes the test pass.
+    //Pseudo-Code for Function:
+        // Function Signature:
+            //Input: a number
+            // Output: Returns a string stating Whether the number is evenly divisble by 3 or not. 
+        //Process:
+            // Create a function called divisibleOrNot taking in a number as an argument, call the parameter number1.
+            // Use a conditional statement (if/else) and return whether the number is divisible by 3 or not. 
+            // Will have to use modules in order to find the divisibility.
+const divisibleOrNot = (number1) => {
+    if(number1 % 3 === 0) {
+        return `${number1} is divisble by three`
+    } else {
+        return `${number1} is not divisble by three`
+    }
+}
 
 
 
 // --------------------2) Create a function that takes in an array of words and returns an array with all the words capitalized.
 
 // a) Create a test with expect statements for each of the variables provided.
 
-const randomNouns1 = ["streetlamp", "potato", "teeth", "conclusion", "nephew"]
+// const randomNouns1 = ["streetlamp", "potato", "teeth", "conclusion", "nephew"]
 // Expected output: ["Streetlamp", "Potato", "Teeth", "Conclusion", "Nephew"]
-const randomNouns2 = ["temperature", "database", "chopsticks", "mango", "deduction"]
+// const randomNouns2 = ["temperature", "database", "chopsticks", "mango", "deduction"]
 // Expected output: ["Temperature", "Database", "Chopsticks", "Mango", "Deduction"]
 
+//Psuedo-Code for Jest Test:
+    // A describe method that lists the name of the function OR naming of the particular test.
+    // A test/it method, nested within the describe block, that in plain words, describe what the function does.
+    // An expect method, nested within the test block, calling on the firstLetterCaps() function, followed by the toEqual() matcher that checks the expected output of the function return.
+    // Make 2 expects for each of the const variables provided. 
+
+describe("firstLetterCaps", () => {
+    const randomNouns1 = ["streetlamp", "potato", "teeth", "conclusion", "nephew"]
+    const randomNouns2 = ["temperature", "database", "chopsticks", "mango", "deduction"]
+    it("returns the first letter of every word in caps", () => {
+        expect(firstLetterCaps(randomNouns1)).toEqual(["Streetlamp", "Potato", "Teeth", "Conclusion", "Nephew"])
+        expect(firstLetterCaps(randomNouns2)).toEqual(["Temperature", "Database", "Chopsticks", "Mango", "Deduction"])
+    })
+})
```

Comment:
Test looks great!   While the variables will work outside the it method, it is best practice to assign variables within the scope you will be using them.